src/handler.ts

```diff
@@ -0,0 +1,40 @@
+import { APIGatewayProxyEvent, Context, APIGatewayProxyResult, Callback } from 'aws-lambda';
+import dotenv from 'dotenv';
+import { awsLambdaReceiver, getLastCommandDetails, getLastResponseMessage } from './app';
+import { AwsEvent } from '@slack/bolt/dist/receivers/AwsLambdaReceiver';
+
+dotenv.config();
+
+export const handler = async (
+  event: APIGatewayProxyEvent,
+  context: Context,
+  callback: Callback
+): Promise<APIGatewayProxyResult> => {
+  console.log('Received event:', JSON.stringify(event, null, 2));
+  
+  try {
+    const awsEvent: AwsEvent = {
+      ...event,
+      multiValueQueryStringParameters: event.multiValueQueryStringParameters || {},
+      multiValueHeaders: event.multiValueHeaders || {},
+    };
+    
+    const handler = await awsLambdaReceiver.start();
+    const result = await handler(awsEvent, context, callback);
+    
+    const commandDetails = getLastCommandDetails();
+    const responseMessage = getLastResponseMessage();
+    
+    if (commandDetails && responseMessage) {
```

Comment:
What is this needed for? Why do we need to export these from the slack app as we don't actually use them for anything.